nm/src/nm.c

```diff
@@ -774,7 +743,7 @@ _j4status_nm_init(J4statusCoreInterface *core)
     context->formats.up_other   = j4status_format_string_parse(format_up_other,   _j4status_nm_format_up_other_tokens,   G_N_ELEMENTS(_j4status_nm_format_up_other_tokens),   J4STATUS_NM_DEFAULT_FORMAT_UP_OTHER,   &context->formats.up_other_tokens);
     context->formats.down_other = j4status_format_string_parse(format_down_other, NULL,                                  0,                                                   J4STATUS_NM_DEFAULT_FORMAT_DOWN_OTHER, &context->formats.down_other_tokens);
 
-    context->nm_client = nm_client_new();
+    context->nm_client = nm_client_new(NULL, NULL); /* TODO: Error handling */
```

Comment:
I’m sure you can do the TODO directly, should not be that hard. :-)
However, please move it somewhere at the top of the function to make cleanup easier.